lib/doh.h

```diff
@@ -88,6 +96,9 @@ int Curl_doh_getsock(struct connectdata *conn, curl_socket_t *socks);
 
 #define DOH_MAX_ADDR 24
 #define DOH_MAX_CNAME 4
+#ifdef USE_HTTPSRR
+#define DOH_MAX_HTTPS 4
+#endif
```

Comment:
This define could be done unconditionally.

Reply:
done